meta-balena-common/recipes-support/hostapp-update-hooks/files/1-bootfiles

```diff
@@ -151,7 +166,7 @@ for filepath in $(find /resin-boot -type f | sed 's#^/resin-boot##g'); do
 done
 info "Boot partition can accomodate the new update."
 
-find -L /mnt/boot/ $(printf "! -name %s " $(for blacklisted_file in  $bootfiles_blacklist; do echo $blacklisted_file | awk -F'/' '{print $NF}'; done)) -exec touch {} +
+find -L "${boot_mountpoint}" $(printf "! -name %s " $(for blacklisted_file in  $bootfiles_blacklist; do echo $blacklisted_file | awk -F'/' '{print $NF}'; done)) -exec touch {} +
```

Comment:
Could we maybe break this up for readability?

Reply:
We could make it more readable but it has nothing to do with the change - I'd rather do it in a different PR to limit the changes of unwanted functional changes.